### ISSUE TITLE
Develop Validation loss metric 

### ISSUE THREAD

**Issue by axiomcura** [WayScience/CytoTrajectory#3]
This looks like it is tracking the training loss, also consider incorporating the validation loss

_Originally posted by @MattsonCam in https://github.com/WayScience/CytoTrajectory/pull/1#discussion_r1373634468_
            

Currently, the code only checks for training loss. This issue focuses in developing a validation loss metric.
